StarCellBio/settings.py

```diff
@@ -40,7 +40,7 @@
 
 # Default logging configuration
 LOG_LEVEL = 'DEBUG'
-DJANGO_LOG_LEVEL = 'DEBUG'
+DJANGO_LOG_LEVEL = 'WARNING'
```

Comment:
It makes development a giant pain, because it prints the sql for every
single django query on page load.

On Tue, Feb 23, 2016 at 8:54 AM, Tobias Macey <notifications@github.com>
wrote:

> In StarCellBio/settings.py
> <https://github.com/starteam/starcellbio_html/pull/593#discussion_r53781709>
> :
>
> > @@ -40,7 +40,7 @@
> >
> >  # Default logging configuration
> >  LOG_LEVEL = 'DEBUG'
> > -DJANGO_LOG_LEVEL = 'DEBUG'
> > +DJANGO_LOG_LEVEL = 'WARNING'
>
> This is safe to leave at debug because it gets overridden during
> deployment by Ansible.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/starteam/starcellbio_html/pull/593/files#r53781709>.
>